### ISSUE TITLE
message - explainer for deletion

### ISSUE THREAD

**Issue by josefkreitmayer** [TransforMap/transformap-editor#25]
A note function for explaining why sthg is deleted

**Comment by species** [comment]
Do you propose something like a 'commit message' that describes what someone has changed and is only informational for admins/developers and not intended for the end user?

**Comment by josefkreitmayer** [comment]
Yes, it is like a commit message from the user, that deletes the initiative, to understand the reason for the deletion, otherwise it seems unclear, why that happened, and can just be taken as a fact. As deletions will not be shown anymore to the user, there could be some kind of deletion log or such once a year, to track the deletions.
